pwa/src/data/years.ts

```diff
@@ -1,114 +1,170 @@
 export const TEMP_YEARS = [

```

Comment:
Is this still temporary? Are we going to receive the options from the backend?

If not, I think we should refactor that. If it _is_ temporary, I think we should create a JIRA issue about it (I haven't seen it).